src/main/java/frc/robot/commands/ExampleCommand.java

```diff
@@ -14,7 +14,7 @@
  * An example command that uses an example subsystem.
  */
 public class ExampleCommand extends CommandBase {
-  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  @SuppressWarnings({ "PMD.UnusedPrivateField", "PMD.SingularField" })
```

Comment:
Unnecessary annotation

Reply:
The curly braces?

Reply:
Never mind. After some quick googling, I now know what an annotation is.